src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -4003,6 +4003,9 @@ int wm_vuldet_oval_xml_parser(OS_XML *xml, XML_NODE node, wm_vuldet_db *parsed_o
         }
         else if ((dist == FEED_UBUNTU || dist == FEED_REDHAT) && !strcmp(node[i]->element, XML_CRITERION)) {
             // To avoid a RHSA with no CVEs
+            if (parsed_oval->vulnerabilities == NULL) {
```

Comment:
Add the same in the `if/else` where reading the references, as coverity is complaining as well there.